### DISCUSSION TITLE
creating custom viewset in rest_framework/viewsets.py other than ModelViewSet

### DISCUSSION THREAD

**Discussion by jelanmathewjames** [encode/django-rest-framework#9052] [Ideas & Suggestions]
creating concrete ViewSets classes like in rest_framework/generics.py.

eg :

class CreateViewSet(mixins.CreateModelMixin, GenericViewSet):
    pass

class ListCreateViewSet(mixins.CreateModelMixin,mixins.ListModelMixin, GenericViewSet):
    pass

class ListViewSet(mixins.ListModelMixin, GenericViewSet):
    pass

class DeleteViewSets(mixins.DestroyModelMixin, GenericViewSet):
    pass

class UpdateViewSets(mixins.UpdateModelMixin, GenericViewSet):
    pass

**Comment by auvipy** [comment]
I really don't understand the question

**Reply by jelanmathewjames** [reply]
i have updated the question
